.travis.yml

```diff
@@ -33,22 +24,29 @@ before_install:
           [[ -z "$(git diff --name-only $TRAVIS_COMMIT_RANGE | grep -vE $DOCS_REGEX)" ]]
           DOCS_ONLY=$?
       fi
-# Test
-script:
-  - |
-    if (( $DOCS_ONLY == 0 )); then
-      echo "Running verify-docs"
-      make verify-docs
-    else
-      echo "Running full build"
-      make verify build build-integration build-e2e test images svcat-all
-    fi
 jobs:
+  fast_finish: true
   include:
-    # Test is implicit from the build matrix
-    # Cross Build
-    - stage: xbuild
-      script: make images-all
+    # CI Build
+    - stage: test
+      script:
+      - |
+        if (( $DOCS_ONLY == 0 )); then
+          echo "Running verify-docs"
+          make verify-docs
+        else
+          echo "Running full build"
+          make verify build build-integration build-e2e test images svcat-all
+        fi
+      env: GO_VERSION=1.9
+    # Cross Build Check
```

Comment:
what's the original issue we had that we're trying to solve? I forgot. s390 failed why?

Reply:
qemu binfmt?

Reply:
The problem being avoided is that this broke but because we realize it because nothing in our CI runs the other arches, until we try to do a release (at which point it's super frustrating to find a bug).